### DISCUSSION TITLE
A I

### DISCUSSION THREAD

**Discussion by DJGHOSTMACHINE** [community/community#67404] [Programming Help]
A I With Ableton License

**Comment by mantasu** [comment]
Is this what you're looking for https://help.ableton.com/hc/en-us/articles/209774145-License-FAQ?

If you mean a license for AI-generated content with [Magenta Studio](https://www.ableton.com/en/blog/magenta-studio-free-ai-tools-ableton-live/), then there really is no license and you should be able to claim the created content as your own. License for AI-generated content, in general, is a debatable topic.
